index.js

```diff
@@ -94,16 +94,13 @@ function renderMedia (file, getElem, cb) {
   }
 
   function renderMediaSource () {
+    var tagName = MEDIASOURCE_VIDEO_EXTS.indexOf(extname) >= 0 ? 'video' : 'audio'
+    
     if (!MediaSource) {
-      return nextTick(cb, new Error(
-        'Video/audio streaming is not supported in your browser. You can still share ' +
-        'or download ' + file.name + ' (once it\'s fully downloaded). Use Chrome for ' +
-        'MediaSource support.'
-      ))
+      // Replace error with useBlobURL for non-MediaSource support
+      useBlobURL()
```

Comment:
Because you're not returning here, this will fall-though and call `useVideosteam()` or `useMediaSource()` which require MediaSource support.